### PR TITLE
Add readme

### PR DESCRIPTION
**Pull request by yamaoto**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+1. Start backend Cynosura.Studio.Web
+2. Start frontend Cynosura.Studio.Web.Ng
+3. In browser create new solution in Solutions section
+
+### Prerequisites
+
+* dot net core 2.2 SDK
+* Angular CLI 7
+* MS SQL Server (you can use docker-compose if haven't)
```
Comment (bacr) [inline README.md:16]: if you don't have it

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+* Not recomended to use recursions. If possible it is better to use loops
+
+## Restrictions
+* Do not use `goto`
+* Do not use LINQ syntax
+
+# ES/TS Code Styling
+
+* Use 4 spaces instead of tab
+* Use double qout(`"`) for literals
```
Comment (bacr) [inline CONTRIBUTING.md]: double quotes

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+* We use semicolon;
+* Prefer to use `const` instead of `let`
+* Do no use `var`
```
Comment (bacr) [inline CONTRIBUTING.md]: Do not

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+Whenever possible, avoid the use of abbreviations. When using, apply the policy, perceiving the abbreviation as one whole word.
+
+### `Task`, `Task <>`
+
+For Task, Task <> methods name must have `Async` postfix.
+
+
+## Language optimization
+
+* Prefer to use Implicit typeing insted Explicit
```
Comment (bacr) [inline CONTRIBUTING.md]: typing instead

**Review by bacr — CHANGES_REQUESTED** [review]
Changes requested

File: README.md (hunk 1 above)
Comment (yamaoto) [reply README.md:16]: Solved

File: CONTRIBUTING.md (hunk 3 above)
Comment (yamaoto) [reply CONTRIBUTING.md]: Done

File: CONTRIBUTING.md (hunk 4 above)
Comment (yamaoto) [reply CONTRIBUTING.md]: Done

File: README.md (hunk 1 above)
Comment (bacr) [reply README.md:16]: Not resolved

**Review by bacr — APPROVED** [review]

**Pull request merged**